docs/assets/postlight.svg

```diff
@@ -1,5 +1,27 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 115 27" width="115" height="27">
-  <title>Postlight</title>
-    <path fill="#6633ff" d="M0 0h11v3H0z"></path>
-    <path fill="#252534" fill-rule="evenodd" d="M97.39 14.567h-4.83V5.565h-3.107v20.77h3.106v-8.998h4.83v9h3.1V5.564h-3.1v9.002zm17.083-8.985v2.77h-3.968v17.983H107.4V8.35h-3.97V5.583h11.043zm-39.41 15.98V10.338c0-3.17 2.302-5.06 5.49-5.06 2.986 0 5.545 1.716 5.545 4.565v2.09h-3.11v-1.8c0-1.307-1.1-2.093-2.435-2.093-1.565 0-2.395 1.018-2.395 2.414v10.963c0 1.483.946 2.384 2.395 2.384 2.314 0 2.79-1.06 2.79-3.31v-1.76H80.58v-2.77h5.52l-.01 10.382h-2.744v-1.39h-.355c-.6 1.385-1.33 1.672-3.28 1.672-2.36 0-4.64-1.745-4.64-5.06zm-6.363 4.73V5.557h3.247V26.29H68.7zm-3.365-2.752v2.77h-9.318V5.537h3.106V23.54h6.212zM52.897 5.566v2.77h-3.97v18h-3.104v-18h-3.97v-2.77h11.044zm-24.425 15.78h2.916c.208 1.104 1.035 2.416 2.49 2.416 1.53 0 2.53-1.006 2.618-2.387.08-1.227-.435-1.976-1.82-3.12l-3.628-2.92c-1.23-.93-2.49-2.31-2.49-5.14 0-2.624 2.056-4.976 5.54-4.976 1.848 0 3.343.65 4.082 1.43.954 1.015 1.346 1.974 1.595 3.99H36.79c-.376-1.708-1.333-2.586-2.637-2.586-1.504 0-2.316 1.01-2.316 2.33 0 .83.292 1.546 1.38 2.474l3.626 2.92c1.886 1.663 2.932 3.136 2.932 5.82 0 2.826-2.133 5.09-5.53 5.09-2.77 0-5.435-1.31-5.773-5.35zm-6.212-.023c0 1.305-.77 2.347-2.398 2.347-1.602 0-2.433-1.042-2.433-2.347V10.38c0-1.305.83-2.288 2.4-2.288 1.51 0 2.43.87 2.43 2.23v11zM19.858 5.177c3.075 0 5.508 1.61 5.508 5.148v10.908c0 3.625-2.433 5.407-5.535 5.407-3.27 0-5.5-1.724-5.5-5.32V10.5c0-3.48 2.378-5.323 5.536-5.323zm-13.384.39c3.403 0 4.914 2.33 4.914 6.175 0 .06 0 .12-.003.18v.834c-.052 3.926-1.63 6.4-5.087 6.4H3.106v7.18H0V5.566h6.474zM8.01 11.92h.003c.004-.035.004-.07.004-.107 0-2.44-.873-3.497-2.502-3.497h-2.41v8.07h2.41c1.542 0 2.47-.983 2.498-3.632l-.003-.834z"></path>
-</svg>
+<svg id="Layer_1" xmlns="http://www.w3.org/2000/svg" data-name="Layer 1" height="20" width="100" viewBox="0 0 73 13" >
+  <defs>
+    <style>
+    </style>
+    <clipPath id="clip-path" transform="translate(0 -0.66)">
+      <rect fill="none" y="0.66" width="72.75" height="12.65"/>
+    </clipPath>
+  </defs>
+  <title>postlight-logo-labs-13-2</title>
```

Comment:
Can we minify this SVG? I like using [SVGOMG](https://jakearchibald.github.io/svgomg/) for this, personally

Reply:
oh yea! good idea